redis-image/src/main/docker/sentinel_check.sh

```diff
@@ -41,18 +23,4 @@ if [[ "$isRunning" == false ]]; then
     exit 1
 fi
 
-getMasterAddress masterAddr
-echo "Master addr: $masterAddr"
-if [[ -z ${masterAddr} ]]; then
-    echo "No master found"
-    exit 1
-fi
-
-isMasterDown masterDown
-echo "Is master down: $masterDown"
-if [[ "$masterDown" == true ]]; then
-    echo "Redis Sentinel is unable to contact Master"
-    exit 1
-fi
```

Comment:
We don't want to take then sentinels down if the master node goes down.  We want them to stay up and perform leader election on remaining nodes.